### ISSUE TITLE
UX and UI issues on real-time map widget

### ISSUE THREAD

**Issue by tassoman** [matomo-org/matomo#15207]
Running matomo 3.13 we have issues on [Earthed display of real-time visits](https://demo.matomo.org/index.php?module=CoreHome&action=index&idSite=62&period=day&date=yesterday#?idSite=62&period=day&date=yesterday&segment=&category=General_Visitors&subcategory=UserCountryMap_RealTimeMap)

- widget's title is missing, [compare to other widgets](https://demo.matomo.org/index.php?module=CoreHome&action=index&idSite=62&period=day&date=yesterday#?idSite=62&period=day&date=yesterday&segment=&category=General_Visitors&subcategory=UserCountry_SubmenuLocations)
- cursor over countries stills default and doesn't change as 👆 pointer, so users can't understand that clicking on the country they can zoom in 🔍 ➕  the view.
- clicking a region (province/state) zooms out wrongly resetting the map. This event should be removed. 
- Any zoom out 🔍 ➖ button is displayed during zoomed navigation
- no wheel scroll event is caught, maybe would be useful as zooming shortcut

**Comment by Findus23** [comment]
I don't think it is worth changing these things now that it is mostly broken anyway.

But these are useful things to keep in mind when building the new map in Matomo 4.

**Comment by sgiehl** [comment]
refs #11929
